server.cc

```diff
@@ -139,16 +151,13 @@ int error_inject(volatile int in_flight, std::string path, std::string method)
         return 0;
     }
 
-    // get the fault injection descritor
+    // get the fault injection descriptor
     auto descr = fault_map[method];
 
-    int err_no = 0;
-
     // get the err_no to inject
-    if (descr.err_no) {
```

Comment:
Let's instead special case the `err_nos.size() == 1` case here.

Although drawing a random number in `dist(0, 1)` is technically correct, there's no reason to waste cpu cycles on the random code if only one `err_no` is used.